config/dev.webpack.config.js

```diff
@@ -4,7 +4,20 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
-    debug: true
+    debug: true,
+    useProxy: process.env.USEPROXY ? true : false,
+    deployment: process.env.BETA ? 'beta/apps' : 'apps',
```

Comment:
This may break your build, you want to keep the deployment `undefined` if not using the beta env variable. Like here: 
https://github.com/RedHatInsights/frontend-starter-app/blob/master/config/dev.webpack.config.js#L8.

Also, you are missing the start:beta command: https://github.com/RedHatInsights/frontend-starter-app/blob/master/package.json#L19

Reply:
As this is only the dev webpack config, build should be fine right?